### PR TITLE
[Data Transfer] Update transfer command for pull transfers

### PR DESCRIPTION
**Pull request by innerdvations**
### What does it do?

Updates the `strapi transfer` command to support the pull (--from) feature

### Why is it needed?

So that transfer from a remote source can be accessed via the strapi CLI.

### How to test it?

Run `strapi transfer --from remote_url`

### CODE REVIEW

File: packages/core/strapi/lib/commands/transfer/transfer.js [hunk 1]
```diff
@@ -46,15 +46,16 @@ module.exports = async (opts) => {
     exitWith(1, 'Could not parse command arguments');
   }
 
-  const strapi = await createStrapiInstance();
+  if (!(opts.from || opts.to) || (opts.from && opts.to)) {
+    console.log('from', opts.from);
+    console.log('to', opts.to);
```
Comment (innerdvations) [inline packages/core/strapi/lib/commands/transfer/transfer.js:51]:
```suggestion
```

File: packages/core/strapi/lib/commands/__tests__/data-transfer/transfer.test.js [hunk 2]
```diff
@@ -139,6 +156,8 @@ describe('Transfer', () => {
     });
   });
 
+  it.todo('uses local Strapi destination when to is not specified');
```
Comment (christiancp100) [inline packages/core/strapi/lib/commands/__tests__/data-transfer/transfer.test.js:159]: Can't be this test be implemented?

**Review by christiancp100 — APPROVED** [review]
LGTM

File: packages/core/strapi/lib/commands/__tests__/data-transfer/transfer.test.js (hunk 2 above)
Comment (innerdvations) [reply packages/core/strapi/lib/commands/__tests__/data-transfer/transfer.test.js:159]: Not until pull is merged :)

We'll update this after your PR is merged. Or add the test to yours directly.

**Pull request merged**